GitSavvy.sublime-settings

```diff
@@ -166,6 +166,18 @@
     "log_show_more_commit_info": true,
 
     /*
+        Diffstat look like
+        core/commands/blame.py     | 13 +++++--------
+        core/commands/log.py       |  9 +++------
+        core/git_mixins/history.py |  8 ++------
+        3 files changed, 10 insertions(+), 20 deletions(-)
+
+        Set to `false` if don't want to show this when view a commit.
+     */
+    "show_diffstat": true,
```

Comment:
I think we can keep it all in one setting since if you are interested in diffstat somewhere you would probably like so see it elsewhere. 
Maybe we can have one setting for all views and one setting to display it in commit view